requirements.txt

```
@@ -10,6 +10,7 @@ charm-lib-node-base @ git+https://github.com/charmed-kubernetes/layer-kubernetes
 charm-lib-reconciler @ git+https://github.com/charmed-kubernetes/charm-lib-reconciler
 cosl == 0.0.7
 gunicorn >= 20.0.0,<21.0.0
+interface_hacluster @ git+https://github.com/openstack/charm-interface-hacluster
```

Let's use the fork since it's there.  We can keep it rev'd by branch

```suggestion
interface_hacluster @ git+https://github.com/charmed-kubernetes/charm-interface-hacluster
```